### PR TITLE
docs: restructure README

### PR DESCRIPTION
**Pull request by jpetazzo**
I rewrote a lot of stuff, so feel free to cherry-pick the parts that you like and ditch the rest :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,91 +1,186 @@
...
+kubectl create namespace kuik-system
+kubectl apply -f /tmp/kuik.yaml --namespace kuik-system
 ```
 
-**From [enix/helm-charts](https://github.com/enix/helm-charts) repository:**
+## Configuration and customization
+
+If you want to change e.g. the expiration delay, the port number used by the proxy, enable persistence (with a PVC) for the registry cache... You can do that with standard Helm values.
+
+Check [values.yaml](/helm/kube-image-keeper/values.yaml) for the full list of parameters, their meaning, and default values.
```
Comment (paullaffitte) [inline README.md:144]: Maybe we should link the values section from the [artifacthub readme](https://artifacthub.io/packages/helm/enix/kube-image-keeper#values) instead  as it is more readable than yaml and it is generated so it's always up to date. Eventually keep both (values.yaml + artifacthub readme)

File: README.md [hunk 2]
```diff
@@ -1,91 +1,186 @@
...
 
-kube-image-keeper (a.k.a. *kuik*) is a container image caching system designed for Kubernetes.
-It ensures the availability of your favorite container images by keeping a local copy within your k8s cluster.  
+kube-image-keeper (a.k.a. *kuik*, which is pronounced /kwɪk/, like "quick") is a container image caching system for Kubernetes.
```
Comment (paullaffitte) [inline README.md:8]: The pronunciation has been moved from the main title to here, @Nicolasgouze what do you think about it?

**Review by paullaffitte — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (jpetazzo) [reply README.md:144]: Good idea; I'll add an extra sentence with a link to the Artifact Hub! 💯

**Review by paullaffitte — APPROVED** [review]

**Pull request merged**